src/components/grid/grid-test.stories.tsx

```diff
@@ -330,3 +329,210 @@ export const GridLayoutExample = () => {
     </GridContainer>
   );
 };
+
+export const CustomPadding = () => {
```

Comment:
comment: we probably need to test a wide range of the permutations that the padding interface can accept here. It might be better to reduce the amount of items and pods here and iterate over an array that sets the various padding props (short and long form) to both numbers between 0 and 8 and valid CSS strings